src/cli-options.ts

```diff
@@ -277,7 +277,11 @@ const cliOptions: CLIOption[] = [
   {
     long: 'cache',
     description: 'Cache versions to the cache file',
-    default: false,
+    type: 'boolean',
+  },
+  {
+    long: 'cacheClear',
+    description: 'Remove the default cache file',
```

Comment:
Let's set the description to `Clears the default cache, or the cache file specified by --cacheFile`.